scripts/common/proxy.sh

```diff
@@ -71,15 +114,28 @@ function configure_proxy() {
         unset HTTPS_PROXY
         return
     fi
-    if [ -z "$PROXY_ADDRESS" ] && [ -z "$ENV_PROXY_ADDRESS" ]; then
+
+    if [ -z "$PROXY_ADDRESS" ] && [ -z "$ENV_HTTP_PROXY_ADDRESS" ] && [ -z "$ENV_HTTPS_PROXY_ADDRESS" ]; then
         log "Not using proxy address"
         return
     fi
-    if [ -z "$PROXY_ADDRESS" ]; then
-        PROXY_ADDRESS="$ENV_PROXY_ADDRESS"
+
+    # if the proxy has been set in the installer we use that one for both
+    # http and https.
+    if [ -n "$PROXY_ADDRESS" ]; then
+        logWarn "Overriding HTTP and HTTPS proxies addresses with $PROXY_ADDRESS"
+        PROXY_HTTPS_ADDRESS="$PROXY_ADDRESS"
+        return
+    fi
+
+    # if user hasn't provide any proxy configuration we use the ones
+    # present in the environment.
+    PROXY_ADDRESS="$ENV_HTTP_PROXY_ADDRESS"
+    PROXY_HTTPS_ADDRESS="$ENV_HTTP_PROXY_ADDRESS"
+    if [ -n "$ENV_HTTPS_PROXY_ADDRESS" ]; then
+        PROXY_HTTPS_ADDRESS="$ENV_HTTPS_PROXY_ADDRESS"
     fi
-    export https_proxy="$PROXY_ADDRESS"
```

Comment:
once again i dont think you can lose this but it may be unnecessary if it is set for all cases above

Reply:
Let's discuss this [here](https://github.com/replicatedhq/kURL/pull/4538#discussion_r1204163307)